nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -0,0 +1,237 @@
+
+    radius: float, optional.
+        Indicates, in millimeters, the radius for the sphere around the seed.
+        Default is None (signal is extracted on a single voxel).
+
+    smoothing_fwhm: float, optional
+        If smoothing_fwhm is not None, it gives the full-width half maximum in
+        millimeters of the spatial smoothing to apply to the signal.
+
+    standardize: boolean, optional
+        If standardize is True, the time-series are centered and normed:
+        their mean is set to 0 and their variance to 1 in the time dimension.
+
+    detrend: boolean, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    low_pass: False or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    high_pass: False or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    t_r: float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    memory: joblib.Memory or str, optional
+        Used to cache the region extraction process.
+        By default, no caching is done. If a string is given, it is the
+        path to the caching directory.
+
+    memory_level: int, optional
+        Aggressiveness of memory caching. The higher the number, the higher
+        the number of functions that will be cached. Zero means no caching.
+
+    verbose: integer, optional
+        Indicate the level of verbosity. By default, nothing is printed.
+
+    See also
+    ========
+    nilearn.input_data.NiftiMasker
+    """
+    # memory and memory_level are used by CacheMixin.
+
+    def __init__(self, seeds, radius=None, smoothing_fwhm=None, standardize=False,
+                 detrend=False, low_pass=None, high_pass=None, t_r=None,
+                 memory=Memory(cachedir=None, verbose=0), memory_level=1,
+                 verbose=0):
+        self.seeds = seeds
+        self.radius = radius
+
+        # Parameters for _smooth_array
+        self.smoothing_fwhm = smoothing_fwhm
+
+        # Parameters for clean()
+        self.standardize = standardize
+        self.detrend = detrend
+        self.low_pass = low_pass
+        self.high_pass = high_pass
+        self.t_r = t_r
+
+        # Parameters for joblib
+        self.memory = memory
+        self.memory_level = memory_level
+
+        self.verbose = verbose
+
+    def fit(self, X=None, y=None):
+        """Prepare signal extraction from regions.
+
+        All parameters are unused, they are for scikit-learn compatibility.
+        """
+        if isinstance(self.seeds, basestring) or is_img(self.seeds):
+            # Take the coordinates of seeds in native space
+            seeds_img = check_niimg(self.seeds, ensure_3d=True)
+            seeds = list(np.where(seeds_img.get_data()))
```

Comment:
No. Do you think that using `masking._load_mask_img` is enough or should we specifically count the number of seeds and raise an error if there are too many?